akka-stream/src/main/scala/akka/stream/io/SslTls.scala

```diff
@@ -380,6 +380,10 @@ sealed trait SslTlsOutbound
  *  - `enabledProtocols` will be passed to `SSLEngine::setEnabledProtocols()`
  *  - `clientAuth` will be passed to `SSLEngine::setWantClientAuth()` or `SSLEngine.setNeedClientAuth()`, respectively
  *  - `sslParameters` will be passed to `SSLEngine::setSSLParameters()`
+ *
+ * Please note that passing `clientAuth = None` means that no change is done
```

Comment:
There seems to be some misunderstanding on this class, even though I think it has an extremely descriptive name: this is for _negotiating a new session_, so setting `None` means “don’t change from how the previous session was negotiated”, JDK defaults don’t come into the picture unless we are talking about the very first session, where these values describe indeed the deviation from just using the SslContext alone.